### PR TITLE
Created tables Order and ItemOrder

### LINKED ISSUE #5: add order table

### PR DESCRIPTION
**Pull request by joaoryff**

### CODE REVIEW

File: database/migrations/2020_02_13_210224_create_item_order_table.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('ItemOrders', function (Blueprint $table) {
+            $table->bigIncrements('id');
```
Comment (MauDevFernandes) [inline database/migrations/2020_02_13_210224_create_item_order_table.php:17]: Good test :)

**Review by MauDevFernandes — APPROVED** [review]
2 out 9 files approved

File: database/migrations/2020_02_13_235911_create_question_pools_table.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('questionPolls', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->timestamps();
```
Comment (LFortunatoC) [inline database/migrations/2020_02_13_235911_create_question_pools_table.php]: It's missing the table fields !!!

File: database/migrations/2020_02_13_234350_create_evaluations_table.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('evaluations', function (Blueprint $table) {
+            $table->bigIncrements('id');
```
Comment (LFortunatoC) [inline database/migrations/2020_02_13_234350_create_evaluations_table.php:17]: It's missing the table fields

**Review by LFortunatoC — CHANGES_REQUESTED** [review]

**Pull request merged**